zulipterminal/config/themes.py

```diff
@@ -189,6 +189,12 @@
          None,             DEF['white'],              DEF['dark_cyan']),
         ('area:error',     'white',                   'dark red',
          None,             DEF['white'],               DEF['dark_red']),
+        ('area:neutral',   'black',                   'light gray',
+         None,             DEF['black'],              DEF['light_gray']),
```

Comment:
We should change this to `task`, to be more consistent with other footer styles. We could always have another one for `area:neutral` if the need be.

Reply:
Thanks for pointing that out. The original intent for it was that it might be used (in general) for other areas (in future) where there's no highlighting required, e.g., typing status. However, since I couldn't think of more such examples, currently we could just align it with other footer-dedicated attributes. We could simply add such themes later in any case :+1: